certified-connectors/App Power Forms/apiDefinition.swagger.json

```diff
@@ -0,0 +1,647 @@
+        "parameters": [
+          {
+            "name": "instanceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "ID of the card to wait for",
+            "x-ms-summary": "Card ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "name",
+            "required": true,
+            "type": "string",
+            "in": "path",
+            "x-ms-summary": "Card name",
+            "x-ms-url-encoding": "single",
+            "description": "Name of the card providing the schema.",
+            "x-ms-dynamic-values": {
+              "operationId": "MetadataListOfCards",
+              "value-path": "name",
+              "value-title": "displayName",
+              "parameters": {
+                "cardType": "ALL"
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/GetResponseSchema/{cardName}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "No need to specify response schema",
+            "schema":{}
+          }
+        },
+        "operationId": "GetResponseSchema",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "cardName",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/MetadataListOfCards/{cardType}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "List of cards",
+            "schema":{
+              "type":"array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string"
+                  },
+                  "displayName":{
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
+        "operationId": "MetadataListOfCards",
+        "parameters": [
+          {
+            "name": "cardType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/MetadataFormFieldTypes": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Form field types",
+            "schema":{
+              "type":"array",
+              "items": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "operationId": "MetadataFormFieldTypes",
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/MetadataFieldTypeSpecificSchema": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "No need",
+            "schema":{}
```

Comment:
@TerhoAntila Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
No parse JSON required here either. This operation is used to display different set of fields based on the field type selection made by the user. See the attached pic.
![image](https://user-images.githubusercontent.com/10154416/152111948-daaaa2c3-e579-469c-bda0-db6477fe3e2b.png)

Reply:
Resolution committed.